dali/base/dautils.cpp

```diff
@@ -427,23 +426,7 @@ void CDfsLogicalFileName::normalizeName(const char *name, StringAttr &res, bool
                     {
                         ep.getUrlStr(str.append("::"));
                         s = ns1;
-                        if (isext)
-                        {
-                            external = true;
-                            if (s[2]=='>')
-                            {
-                                str.append("::");
```

Comment:
I'm not sure what this code is here for, but I don't see a counterpart in the new code.

Some strange processing of file::ip::>blah

which I think will have almost no effect except for setting tailpos earlier if blah contains ::.